app/models/oclc/lc_call_slips/keyword_field.rb

```diff
@@ -5,6 +5,11 @@ module LcCallSlips
     # Marc::DataField for the keywords that a selector
     # is interested in.
     class KeywordField
+      extend FFI::Library
+      c_lib_extension = /darwin/.match?(RUBY_PLATFORM) ? 'dylib' : 'so'
+      ffi_lib Rails.root.join('lib_jobs_rs', 'target', 'release', "liblib_jobs.#{c_lib_extension}")
+      attach_function :normalize_keyword, [:string], :strptr
```

Comment:
`normalize_keyword` is the name of our rust function, and we specify that it takes a string and returns a string pointer.